Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs

```diff
@@ -1597,6 +1660,19 @@ public static void updateMerchantDetailsResponse(updateMerchantDetailsResponse a
             {
             }
         }
+        public static void getCustomerPaymentProfileNonceRequest(getCustomerPaymentProfileNonceRequest argument)
+        {
+            if (null != argument)
+            {
+            }
+        }
```

Comment:
This is redundant. Add some code here or remove the if block.

Reply:
Removed the IF block.